iotedgedev/modules.py

```diff
@@ -145,8 +145,11 @@ def build_push(self, template_file, default_platform, no_build=False, no_push=Fa
         modules_path = os.path.join(template_file_folder, self.envvars.MODULES_PATH)
         if os.path.isdir(modules_path):
             for folder_name in os.listdir(modules_path):
-                module = Module(self.envvars, self.utility, os.path.join(modules_path, folder_name))
-                self._update_module_maps("MODULES.{0}".format(folder_name), module, placeholder_tag_map, tag_build_profile_map, default_platform)
+                try:
+                    module = Module(self.envvars, self.utility, os.path.join(modules_path, folder_name))
+                    self._update_module_maps("MODULES.{0}".format(folder_name), module, placeholder_tag_map, tag_build_profile_map, default_platform)
+                except FileNotFoundError:
```

Comment:
we also need handle even module.json is found but the schema is not correct. I think it should not impact the processing of other modules.

Reply:
Current behavior should aligns with the behavior in v1.x version. What's the expected behavior when the schema is not correct?

Reply:
As discussed, just check the file path instead of catching exception for performance. And we will keep current behavior for now.